proposed/2023/sponsor-link.md

```diff
@@ -0,0 +1,164 @@
+# Sponsor links for NuGet packages
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Jon Douglas <!-- GitHub username link -->
+- [#9339](https://github.com/NuGet/NuGetGallery/issues/9339) <!-- GitHub Issue link -->
+
+## Summary
+
+<!-- One-paragraph description of the proposal. -->
+
+Today, more than 9/10 companies use open source software in at least some capacity. One key concern is ensuring that open source is financially sustainable. More specifically, how do we make sure the developers maintaining some of the most critical software being used by these companies, governments, nonprofits, etc are able to keep doing their important work?
+
+This proposal introduces an end-to-end experience of bringing visibility to the sponsorship requirements of open source packages on the NuGet.org registry and make NuGet tooling aware of it. This is to ensure that the sponsorship requirements make it down to the developers who are using open source software in a non-disruptive way.
+
+Package authors will be able to express their sponsorship needs through new `sponsor` package metadata specifying a popular verified funding mechanism (GitHub Sponsors, Patreon, etc). This may consist of a list of reputable funding providers such as patreon, tidelift, github sponsors, and the relative link to each.
```

Comment:
```suggestion
Package authors will be able to express their sponsorship needs through new `sponsor` package metadata specifying a popular verified funding mechanism (GitHub Sponsors, Patreon, etc). This may consist of a list of reputable funding providers such as patreon, tidelift, github sponsors, and the respective link to each.
```

Relative link is a bit confusing :)

Reply:
Thank you, will add!